### DISCUSSION TITLE
What is the equivalent in python of SetTTFName?

### DISCUSSION THREAD

**Discussion by GlassGruber** [fontforge/fontforge#5200] [Q&A]
Hey there I'm trying to figure out what is the python scripting equivalent of `SetTTFName` function.

Specifically I'm trying to reset the font unique id field, with native scripting I would do this `SetTTFName(0x409,3,"")`. 

By the way where the links in the documentation for [SetTTFName](https://fontforge.org/docs/scripting/scripting-alpha.html#SetTTFName) are broken, below are working links to MS docs

- https://learn.microsoft.com/en-us/typography/opentype/spec/name#name-ids
- https://learn.microsoft.com/en-us/typography/opentype/spec/name#windows-language-ids

Thank you

**Comment by gitteric** [comment]
Late, but incase someone else stumbles upon it:
Check out:
- [font.sfnt_names](https://fontforge.org/docs/scripting/python/fontforge.html#fontforge.font.sfnt_names)
- [font.appendSFNTName](https://fontforge.org/docs/scripting/python/fontforge.html#fontforge.font.appendSFNTName)
